.github/workflows/superlinter.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
```

Comment:
Dałoby radę dodać badge do readme, żeby było widać, czy build failuje?

Reply:
Czemu  ten workflow nazyw się superlinter? superlinter to nazwa jakiegoś projektu do sprawdzania składni w plikach. nazwij go jakoś build.yml albo maven.yml albo diary.yml

Reply:
done